packages/upswyng-server/EVENTLOG.md

```diff
@@ -0,0 +1,154 @@
+# Event Logs
+
+UpSwyng server has a framework used to record business events happening in the system
+like Resources being created or Alerts going live. Events are recorded in the database
+and are shown in the UI accessable by administrators. When an event is recorded the
+`upswyngbot` on Slack can also send a message to its group.
+
+**See [PR #326](https://github.com/CodeForBoulder/upswyng/pull/326) for the implementation of the steps in this document**
+
+## Creating a new Event Log kind
+
+For this example, we'll setup logging for when an Alert goes live. The worker job checks regularly
```

Comment:
Definitely separate from event docs, but scheduled workers may be a good thing to have an overview of as well.